extensions/transferto/action.go

```diff
@@ -49,18 +49,18 @@ func (a *TransferAirtimeAction) AllowedFlowTypes() []flows.FlowType {
 }
 
 // Execute runs this action
-func (a *TransferAirtimeAction) Execute(run flows.FlowRun, step flows.Step, log flows.EventLog) error {
+func (a *TransferAirtimeAction) Execute(run flows.FlowRun, step flows.Step) error {
```

Comment:
now that events don't do anything there's no need for this model of applying them after actions, so we don't need the whole event log thing - actions just log events directly on the run